examples/Headings/PageTitle.md

```diff
@@ -0,0 +1,4 @@
+A top level heading, used for page headers, application titles etc.
```

Comment:
I think we need a note here specifying that there should only be one `<PageTitle>` element on any "page" within the app...

Reply:
Good idea.

Reply:
Addressed in 4d138aac3fe5c71f72a5acefe9f243d938445a86